components/generic/Header/Header.tsx

```diff
@@ -0,0 +1,44 @@
+import { useCallback, useState } from 'react';
+import { clsx as cx } from 'clsx';
+
+import { Container } from '../Container/Container';
+import { MenuIcon } from '../Icons/MenuIcon';
+import { NotificationsIcon } from '../Icons/NotificationsIcon';
+import { AccountIcon } from '../Icons/AccountIcon';
+import { HeaderLink } from './HeaderLink';
+
+export const Header = () => {
+  const [mobileMenuOpened, setMobileMenuOpened] = useState(false);
+  const toggleMobileMenu = useCallback(() => setMobileMenuOpened((p) => !p), []);
+
+  return (
+    <header className="bg-gray-50">
+      <Container
+        className="py-4 sm:py-5 lg:py-6 sm:flex sm:flex-row-reverse sm:justify-between"
+        as="nav"
+      >
+        <div className="w-full flex justify-between items-center sm:w-fit">
+          <button className="sm:hidden" onClick={toggleMobileMenu}>
+            <MenuIcon />
+            <span className="sr-only">Otwórz menu nawigacyjne strony.</span>
+          </button>
+          <div className="flex gap-3">
+            <NotificationsIcon className="lg:w-8 lg:h-8" />
+            <AccountIcon className="lg:w-8 lg:h-8" />
```

Comment:
I think we can remove width/height from the Icon component and pass it as CSS, to unify it.